client/src/components/NationwideMap.vue

```diff
@@ -67,13 +66,30 @@ export default defineComponent({
         '--height': this.height,
       };
     },
+    currentDataLayerId() {
+      return this.mapState?.mapData?.currentDataLayerId;
+    },
+    visibleLayer() {
+      return this.possibleLayers?.find(l =>
+        this.map?.getLayer(l.styleLayer.id) != null &&
```

Comment:
why do we need this check?

Reply:
Looking below, is this because we can have layers in ALL_DATA_LAYERS which aren't in the map?

Reply:
If there's ever a state where the map does not have anything visible, then it should check the currentDataLayer